docs/configuration.rst

```diff
@@ -923,6 +923,20 @@ Mapping property: ``schema_file``
 .. |Built-in schema| replace:: *Built-in schema*
 .. _Built-in schema: https://github.com/divolte/divolte-schema
 
+Mapping property: ``schema_id``
+"""""""""""""""""""""""""""""""
+:Description:
+  The avro records written to Kafka are 'naked' by default.  Which schema was used to write them is therefore not clear from the message itself.  When the schema evolves over time, such metadata is necessary to be able to read the records.  The ``schema_id`` allows the avro serializer to prepend each record with this id.  This becomes effective when the kafka sink mode is ``confluent``, for example.
+:Default:
+    *Not specified*
+:Example:
+
+    .. code-block:: none
+
+    divolte.mappings.a_mapping {
+      schema_id = 1234
+    }
```

Comment:
@fvlankvelt so I think we'll need a separate id for the key schema and the value schema, since they will be different.